Cargo.toml

```diff
@@ -122,3 +123,5 @@ tokio-stream = { git = "https://github.com/madsim-rs/tokio.git", rev = "ab251ad"
 tokio-retry = { git = "https://github.com/madsim-rs/rust-tokio-retry.git", rev = "95e2fd3" }
 tokio-postgres = { git = "https://github.com/madsim-rs/rust-postgres.git", rev = "4538cd6" }
 madsim-rdkafka = { git = "https://github.com/madsim-rs/madsim.git", rev = "6d342a9" }
+# madsim-aws-sdk-s3 = { path = "../madsim/madsim-aws-sdk-s3" }
```

Comment:
remove this?

Reply:
The adaptation of madsim is in progress. It will be removed when done.